ledger/accountdb.go

```diff
@@ -1496,19 +1601,16 @@ func performResourceTableMigration(ctx context.Context, tx *sql.Tx, log func(pro
 		if rowsAffected != 1 {
 			return fmt.Errorf("number of affected rows is not 1 - %d", rowsAffected)
 		}
-		rowid, err = insertRes.LastInsertId()
+		rowID, err = insertRes.LastInsertId()
```

Comment:
changed rowID to rowid because the latter is used everywhere else

Reply:
one of the linters want the ID to be written with capital letters..